modules/ROOT/pages/plugins/opensource/image.adoc

```diff
@@ -36,11 +36,11 @@ These configuration options affect the execution of the `image` plugin. Many of
 If you wish to align the image, you can also use the text align buttons while images are selected.
 
 :includedSection: imagePlugin
-#include::partial$configuration/a11y_advanced_options.adoc[]
+include::partial$configuration/a11y_advanced_options.adoc[]
```

Comment:
The `#` or `=` before these includes was important as it lowered the level used for them. Since this was lost the ToC is now incorrect (see all the options use the wrong heading level and are top level entities). As such you need to add `[leveloffset=+1]` to each include to make them use the correct heading level.
![image](https://user-images.githubusercontent.com/1575550/160512891-315285df-5eb4-488e-8164-97b45d11fd22.png)

Reply:
Done

Reply:
Done